packages/eslint-plugin/docs/rules/no-confusing-void-expression.md

```diff
@@ -109,6 +109,27 @@ function doSomething() {
 console.log(void alert('Hello, world!'));
 ```
 
+### `ignoreVoidInVoid`
+
+Allow using `void` type expressions in return value of a function that specified as `void`
+
+Examples of additional **correct** code with this option enabled:
+
+```ts option='{ "ignoreVoidInVoid": true }' showPlaygroundButton
+// type is correct with void, so not evoke error
```

Comment:
[Docs] I'm not sure I got the meaning of those comments right :smile:

"type is correct with void" - the point of this option is to ignore void <-> void cases, type correctness is checked by `tsc`, so I think that we should rephrase these comments or remove them. Furthermore, I think this code is self-explanatory, so I think it would be easy to guess what this option does just by looking at the description and the code. Please correct me if I'm wrong

Reply:
The reason for including the comment was because the code in the previous example included the comment, but there is no other reason. I will remove the comment!